wolfssl/wolfcrypt/port/Renesas/renesas-sce-crypt.h

```diff
@@ -48,17 +68,34 @@ typedef struct tagUser_SCEPKCbInfo {
     uint8_t     sce_clientRandom[SCE_TLS_CLIENTRANDOM_SZ];
     uint8_t     sce_serverRandom[SCE_TLS_SERVERRANDOM_SZ];
     uint8_t     sce_cipher;
+
+   #endif
     
     /* installed key handling */
-    sce_aes_wrapped_key_t   sce_wrapped_key_aes256;
-    uint8_t aes256_installedkey_set:1;
-    sce_aes_wrapped_key_t   sce_wrapped_key_aes128;
-    uint8_t aes128_installedkey_set:1;
+    /* aes */
+    renesas_sce_wrappedkey   sce_wrapped_key_aes256;
+    renesas_sce_wrappedkey   sce_wrapped_key_aes128;
+    
+   #if defined(WOLFSSL_RENESAS_SCEPROTECT_CRYPTONLY)
+    /* rsa */
+    renesas_sce_wrappedkey   sce_wrapped_key_rsapri2048;
+    renesas_sce_wrappedkey   sce_wrapped_key_rsapub2048;
+    renesas_sce_wrappedkey   sce_wrapped_key_rsapri1024;
+    renesas_sce_wrappedkey   sce_wrapped_key_rsapub1024;
+   #endif
     
+    /* key status flags */
     /* flag whether encrypted ec key is set */
-    uint8_t pk_key_set:1;
-    uint8_t session_key_set:1;
-
+    union {
+        uint8_t chr;
+        struct sce_flags1 bits;
+    } flags1;
+    
+    union {
+        uint8_t chr;
+        struct sce_flags2 bits;
+    } flags2
```

Comment:
Same comment applies as above. Should this be something more descriptive than `flags2`?

Reply:
Agree to you.  How about `keyflgs_crypt`?